PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -532,6 +532,50 @@ LRESULT ScintillaEditView::scintillaNew_Proc(HWND hwnd, UINT Message, WPARAM wPa
 						                                              // https://sourceforge.net/p/scintilla/bugs/2412/
 						break;
 
+					case VK_DELETE:
+					{
+						SHORT ctrl = GetKeyState(VK_CONTROL);
+						SHORT alt = GetKeyState(VK_MENU);
+						SHORT shift = GetKeyState(VK_SHIFT);
+						
+						if (!(shift & 0x8000) && !(ctrl & 0x8000) && !(alt & 0x8000)) // DEL + Column-edit
+						{
+							size_t nbSelections = execute(SCI_GETSELECTIONS);
+							if (nbSelections > 1)
+							{
+								execute(SCI_BEGINUNDOACTION);
+								for (size_t i = 0; i < nbSelections; ++i)
+								{
+									LRESULT posStart = execute(SCI_GETSELECTIONNSTART, i);
+									LRESULT posEnd = execute(SCI_GETSELECTIONNEND, i);
+									if (posStart != posEnd)
+									{
+										replaceTarget(L"", posStart, posEnd);
+									}
+									else // posStart == posEnd)
+									{
+										char eolStr[3];
+										Sci_TextRange tr;

```

Comment:
@donho Shouldn't be Sci_TextRangeFull used instead of deprecated Sci_TextRange? Together with the corresponding SCI_GETTEXTRANGEFULL.

Reply:
@chcg 
You are absolutely right!
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/8ae049afbcfa4d37ec5abd81e2df8bf92af1b7d0